### PR TITLE
LPS-183649 Integration Test for LPS-179030

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-183648](https://issues.liferay.com/browse/LPS-183648)

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 1]
```diff
@@ -1317,6 +1318,27 @@ public void testUpdateCustomObjectField() throws Exception {
...
+		LocaleUtil.setDefault(
+			LocaleUtil.GERMANY.getLanguage(), LocaleUtil.GERMANY.getCountry(),
+			LocaleUtil.GERMANY.getVariant());
+
+		objectField = _objectFieldLocalService.updateCustomObjectField(
+			StringPool.BLANK, objectField.getObjectFieldId(), 0,
+			ObjectFieldConstants.BUSINESS_TYPE_INTEGER,
+			ObjectFieldConstants.DB_TYPE_INTEGER, false, true, StringPool.BLANK,
+			objectField.getLabelMap(), false, objectField.getName(), false,
+			false, objectField.getObjectFieldSettings());
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:1333]: @pedro-oliveira446 please check if you can use the existing method `_updateCustomObjectField`:

```
objectField = _updateCustomObjectField(objectField);
```

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 2]
```diff
@@ -1317,6 +1318,27 @@ public void testUpdateCustomObjectField() throws Exception {
...
+
+		Assert.assertNotNull(objectField);
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:1335]: @pedro-oliveira446 I think we can remove this `assert` here. It doesn't seem to add much value to the test to check if the objectField is not null.

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:forward:force
